python/ray/tune/suggest/bayesopt.py

```diff
@@ -88,6 +103,14 @@ def __init__(self,
             random_search_steps (int): Number of initial random searches.
                 This is necessary to avoid initial local overfitting
                 of the Bayesian process.
+            patience (int): Must be > 0. If the optimizer suggests a set of
+                hyperparameters more than 'patience' times,
+                then the whole experiment will stop.
+            skip_duplicate (bool): If true, BayesOptSearch will not create
+                a trial with a previously seen set of hyperparameters. By
+                default, floating values will be reduced to a digit precision
+                of 5. You can override this by setting
+                ``searcher.repeat_float_precision``.
             analysis (ExperimentAnalysis): Optionally, the previous analysis
                 to integrate.
             verbose (int): Sets verbosity level for BayesOpt packages.
```

Comment:
The argument docs are in the wrong order. I'd suggest to either make `verbose` the last argument or move the docstring before `patience`.